### ISSUE TITLE
[Question] Why is the Adam optimizer set with lr_schedule(1)?

### ISSUE THREAD

**Issue by Waterkin** [DLR-RM/stable-baselines3#771]
**Important Note: We do not do technical support, nor consulting** and don't answer personal questions per email.
Please post your question on the [RL Discord](https://discord.com/invite/xhfNqQv), [Reddit](https://www.reddit.com/r/reinforcementlearning/) or [Stack Overflow](https://stackoverflow.com/) in that case.

### Question

In stablebaselines3.common.policies.py, I found in ActorCriticPolicy._build()
`
# Setup optimizer with initial learning rate
self.optimizer = self.optimizer_class(self.parameters(), lr=lr_schedule(1), **self.optimizer_kwargs)
`
Is the lr_schedule(1) means that the learning rate is set as a constant 1, or is it linearly decayed, there is no document about it. Thanks!

### Additional context

Add any other context about the question here.

### Checklist

- [ ] I have read the [documentation](https://stable-baselines3.readthedocs.io/en/master/) (**required**)
- [ ] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo (**required**)

**Comment by Miffyli** [comment]
`lr_schedule` is a function that returns learning rate for inputs in interval [0.0, 1.0], where 1.0 refers to "start of training" and 0.0 "end of training". `lr=lr_schedule(1)` means the initial learning rate is whatever the `lr_schedule` sets in the beginning. If you learning rate argument to the agent class is a constant, then yes, the learning rate is a constant over whole time. See [this example](https://stable-baselines3.readthedocs.io/en/master/guide/examples.html#learning-rate-schedule) for usage.

Feel free to close issue if you have no further uncertainties.

**Comment by Waterkin** [comment]
Thanks for your reply, Miffyli.
